pages/rotas.vue

```diff
@@ -84,15 +102,16 @@ export default {
   methods: {
     getLineHours (lineHours) {
       return lineHours.map(date =>
-        this.formatHour(new Date(date).toTimeString())
+        this.formatHour(new Date(date).toUTCString())
       ).join(' - ')
     },
 
     formatHour (hour) {
-      // Input example: 21:50:00 GMT-0200 (Brasilia Summer Time)
-      // Output: 21:50
+      // Input example: Fri, 01 Jan 1999 14:16:00 GMT
+      // Output: 14:16
+      // After slit, -2 gets the penultimate value
 
-      return hour.split(' ')[0]
+      return hour.split(' ').at(-2)
```

Comment:
tive que mudar pra forçar no formato UTC pq tava quebrando nos testes do github 😬

Reply:
Acho que isso aqui é só preciosismo mesmo, mas na página de home por exemplo a gente usa o `isoString()`, não lembro exatamente onde eu vi mas o pessoal diz que é melhor usar esse cara.
Na home a gente acabou usando isso:
`https://github.com/UnifespCodeLab/emtu-web/blob/46906783655511c37f7d62d9581f6eb6945ce4c1/pages/index.vue#L112`

Reply:
ISO já era o formato que a API retornava, simplifiquei a lógica usando só substring mesmo 😬